netbox/ipam/api/serializers.py

```diff
@@ -365,6 +365,21 @@ def to_representation(self, instance):
         ])
 
 
+class RequestAvailableIPSerializer(IPAddressSerializer):
+    """
+    Representation of a request for a new available Ip
+    """
+    mask_length = serializers.IntegerField(required=False, default=32)
```

Comment:
default=32 ? what about IPv6 ?

Reply:
there are some validation checks done on the `PrefixLengthSerializer`, I guess you can get some inspiration from what has been done here:
```
class PrefixLengthSerializer(serializers.Serializer):

    prefix_length = serializers.IntegerField()

    def to_internal_value(self, data):
        requested_prefix = data.get('prefix_length')
        if requested_prefix is None:
            raise serializers.ValidationError({
                'prefix_length': 'this field can not be missing'
            })
        if not isinstance(requested_prefix, int):
            raise serializers.ValidationError({
                'prefix_length': 'this field must be int type'
            })

        prefix = self.context.get('prefix')
        if prefix.family == 4 and requested_prefix > 32:
            raise serializers.ValidationError({
                'prefix_length': 'Invalid prefix length ({}) for IPv4'.format((requested_prefix))
            })
        elif prefix.family == 6 and requested_prefix > 128:
            raise serializers.ValidationError({
                'prefix_length': 'Invalid prefix length ({}) for IPv6'.format((requested_prefix))
            })
        return data
```

Reply:
I remove the default.
we don't check, as it's not checked to for the gui